automation/release.sh

```diff
@@ -37,7 +37,7 @@ function build_release_artifacts() {
     make olm-verify
     make prom-rules-verify
 
-    docker login -u="$QUAY_USER" -p="$QUAY_PASSWORD" quay.io
+    docker login --username "$(<"$QUAY_USER")" --password-stdin=true quay.io <"$QUAY_PASSWORD"
```

Comment:
This is very `prow` specific. I would prefer if you just completely remove the login here and do it in the corresponding prowjob. The script, while run in CI normally, should also work locally. The login mechanism should not be part of the scripts, since that makes also prow job evolution harder.

Reply:
Good point!